h2o-core/src/main/java/hex/Model.java

```diff
@@ -2997,4 +2997,8 @@ protected boolean isFeatureUsedInPredict(int featureIdx) {
   public boolean isDistributionHuber() {
     return _parms._distribution == DistributionFamily.huber;
   }
+
+  public TwoDimTable[][] getFeatureInteractionsTable(int maxInteractionDepth, int maxTreeDepth, int maxDeepening) {
+    return null;
```

Comment:
FeatureInteractionsCollector is in h2o-algos and it has `Map<String, FeatureInteraction> getFeatureInteractions(...`  declared (so it can't be moved to core), not `TwoDimTable[][] getFeatureInteractionsTable(...`, so it can't be used like this. I will use H2O.unimpl()

Reply:
Then the FeatureInteractionsCollector ifc serves no purpose and can be removed

any reason it is in h2o-algos? I think it would make sense to either move everything to core or to algos so the code can be cleaner

Reply:
As a next step, I will do the same also for gbm but the calculation will be different. Thats why I did FeatureInteractionsCollector, but can remove now. Not sure about moving everything to core, the topic seems to me h2o-algo like, thats why I put it to algo. I can create interface in core so that `Model.java#getFeatureInteractionsTable(.. `is not needed